src/pkg/packager/prepare.go

```diff
@@ -209,7 +209,8 @@ func (p *Packager) findImages(ctx context.Context) (map[string][]string, error)
 			for idx, k := range manifest.Kustomizations {
 				// Generate manifests from kustomizations and place in the package
 				kname := fmt.Sprintf("kustomization-%s-%d.yaml", manifest.Name, idx)
-				destination := filepath.Join(componentPaths.Manifests, kname)
+				// Use the temp folder because "helpers.CreatePathAndCopy" will with the same path will result in the file being empty
```

Comment:
Just realized grammatical error in my comment, I will update it as soon as I can